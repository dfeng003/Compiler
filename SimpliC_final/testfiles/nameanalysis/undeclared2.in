int gcd1(int a, int b) {
while (a != c) {
a = a - b;
}
print();
read();
return gcd2(a, b);
}
