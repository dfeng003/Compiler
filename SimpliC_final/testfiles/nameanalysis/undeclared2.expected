Error at line 2: symbol 'c' is not declared
Error at line 5: the calling of symbol 'print' is wrong
Error at line 7: symbol 'gcd2' is not declared
Error at line 7: the calling of symbol 'gcd2' is wrong
